Mathlib/GroupTheory/Abelianization.lean

```diff
@@ -307,3 +322,80 @@ instance [Finite (commutatorSet G)] :
   exact Finite.card_pos.ne'
 
 end commutatorRepresentatives
+
+/- # Subgroups containing commutator -/
+
+variable {G}
+
+/-- If G and H have multiplications *
+and φ : G → H is a surjective multiplicative map,
+and if G is commutative, then H is commutative.
+
+Since I only use it with groups,
```

Comment:
This part of the comment has probably been forgotten.

Reply:
I deleted the final part of the comment.